src/ProgressOnderwijsUtils/DistinctArray.cs

```diff
@@ -73,6 +73,19 @@ public int Count
         public T this[int index]
             => UnderlyingArrayThatShouldNeverBeMutated()[index];
 
+        /// <inheritdoc />
+        public override bool Equals(object? obj)
+            => obj is DistinctArray<T> other && this == other;
+
+        public override int GetHashCode()
+            => UnderlyingArrayThatShouldNeverBeMutated().GetHashCode();
+
+        public static bool operator ==(DistinctArray<T> a, DistinctArray<T> b)
+            => a.UnderlyingArrayThatShouldNeverBeMutated() == b.UnderlyingArrayThatShouldNeverBeMutated();

```

Comment:
Kan je het niet andersom doen? Dus in de operator de == aanroepen? Dat is tenminste hoe ik het zou verwachten en we, zover ik weet, ook eigenlijk altijd zo doen.

Reply:
dat is meer code, want dit implementeert geen IEquatable, want er was geen accute reden voor.  Wil je dat wel?

Reply:
Vind ik zelf wel zo consequent ja.

Reply:
ok.